docs/source/workflows/predictors.rst

```diff
@@ -22,10 +22,18 @@ There must be at least one input and at least one output.
 Unlike the `SimpleMLPredictor <#simple-ml-predictor>`__, only one model is trained from inputs to the outputs.
 
 AutoMLPredictors support both regression and classification.
-If the output descriptor is a :class:`~citrine.informatics.descriptors.RealDescriptor`, regression is performed.
-If the output descriptor is a :class:`~citrine.informatics.descriptors.CategoricalDescriptor`, classification is performed.
+For each :class:`~citrine.informatics.descriptors.RealDescriptor` output, regression is performed.
+For each :class:`~citrine.informatics.descriptors.CategoricalDescriptor` output, classification is performed.
 Both binary and multi-class classification are supported automatically based on the number of categories in the data.
 
+Assuming they have the same inputs, it is generally preferable to combine multiple related outputs into a single Auto ML Predictor.
+This has several benefits when compared to creating one Auto ML Predictor for each output.
+
+* Higher model accuracy (especially if the data are sparse)
+* Faster training and prediction
+* More accurate uncertainty estimates that understand the correlations between outputs
+* More accurate scoring and candidate suggestions when running a :doc:`design workflow <design_workflows>`.
```

Comment:
Possibly remove the terminal period here, or add a period to each bullet for consistency?